src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,56 @@
+package racingcar;
+
+import static racingcar.CarConstant.*;
+
+public class Car {
+    private final String carName;
+    private String moveString;
+    private int position = INIT_POSITION;
+
+    public Car(String carName) {
+        validateCarName(carName);
+        this.carName = carName;
+    }
+
+    private void validateCarName(String carNameStr) {
+        String[] carNames = carNameStr.split(",");
+
+        for(String carName : carNames){
+            if (carName.trim().length() > MAX_NAME_LENGTH) {
+                throw new IllegalArgumentException("자동차 이름이 5자 초과입니다");
+            }
+        }
+    }
```

Comment:
```suggestion
    private void validateCarName(String carName) {
        if (carName.trim().length() > MAX_NAME_LENGTH) {
            throw new IllegalArgumentException("자동차 이름이 5자 초과입니다");
        }
    }
```
`Car`생성자에 split된 값이 넘어옵니다. `자동차이름`만 체크하면 될 것 같습니다.